### DISCUSSION TITLE
Log files for booting

### DISCUSSION THREAD

**Discussion by Dj-Polyester** [davmac314/dinit#362] [Q&A] [answered]
Hi. When I had problems with systemd, I used journalctl to see the logs for my previous boot. Recently, my computer does not turn on (just outputs status messages) after a suspend. Any ideas on where I should look? Thanks.

**Comment by mobin-2008** [comment]
Hi.

What's inside of `/var/log/` directory. syslogs usually puts the system log in that directory.

**Comment by Dj-Polyester** [comment]
`/var/log`:
```
Xorg.0.log  Xorg.0.log.old  Xorg.1.log  Xorg.2.log  audit  btmp  dinit  lightdm  nvidia-installer.log  old  pacman.PackageKit.log  pacman.log  sddm.log  wtmp
```
`/var/log/dinit`:
```
bluetoothd.log  dbus-daemon.log  lightdm.log
```

**Reply by mobin-2008** [reply]
What's your system distro? It's looks like Artix Linux.

~These logs are for daemons which usually aren't useful for suspend debug. What's inside `/var/log/` directory? not `/var/log/dinit/`.~

**Reply by mobin-2008** [reply]
Artix Linux follows [Arch Linux](https://wiki.archlinux.org/title/Arch_Linux), That means the `syslog` is not pre-installed. Just choose a `syslog` implementation. I use `syslog-ng` for daily use because it works well (at least for me):

1. Install `syslog-ng`:
    ```
    sudo pacman -S syslog-ng syslog-ng-dinit
    ```

2. Enable the `syslog-ng`
    ```
    sudo dinitctl enable syslog-ng
    ```
3. Now you have the system log in your `/var/log/` (Look at the new files like `messages`, `daemon.log` etc)

**Comment by Dj-Polyester** [comment]
Yes it is Artix. After installing `syslog-ng` as you described, `/var/log`:
```
Xorg.0.log      Xorg.1.log  audit     btmp        dinit           lightdm               old                    pacman.log  syslog.log
Xorg.0.log.old  Xorg.2.log  auth.log  daemon.log  everything.log  nvidia-installer.log  pacman.PackageKit.log  sddm.log    wtmp
```
I am puzzled as 5 new files added. Which file should I look when I have a problem with the last boot? I am sorry as I am a newbie, and learning about init systems. Thank you the reply.

**Reply by mobin-2008** [reply, marked as answer]
Restart your system, You should have a new file called `kernel.log`. This file contains the kernel log for current boot and past boots. It's a good start for debugging suspend issues.

**Comment by davmac314** [comment]
Note that during resume logs might not be recording.  So there may be nothing useful after you reboot from a failed resume.

You mentioned:

> my computer does not turn on (just outputs status messages)

I don't suppose you read the status messages to see if there is anything useful there? (Where do these show up btw? Are you on the console and not in a graphical session?)

**Reply by davmac314** [reply]
(I would guess the issue is kernel-related. If it was working earlier, try to roll back to an earlier kernel version).
